src/LabelPool.stories.tsx

```diff
@@ -137,6 +141,12 @@ function BasicTemplate({
     setLabel(newLabel);
 
     storyScene.scene.add(newLabel);
+    const defaultLabel = storyScene.labelPool.acquire();
+    // test transparency
+    defaultLabel.setText("Default");
+    defaultLabel.position.set(0, 0, -1);
+    storyScene.scene.add(defaultLabel);
```

Comment:
The stories look weird now... maybe add a separate story to demonstrate transparency?